test/abbreviation_bug_test.py

```diff
@@ -4,7 +4,7 @@
 sys.path.insert(0, '..')
 
 from versioningDB import versioning
-from pyspatialite import dbapi2
+from sqlite3 import dbapi2
```

Comment:
We never need pyspatialitis again? It is during the creation of the base, via ogr2ogr, that the spatial layer is created? As a precaution, have you checked that the use of a space base works independently of the plugin (normally yes, but just in case).

Reply:
Yes pyspatialite is useless according to [this](https://groups.google.com/forum/#!topic/spatialite-users/o0jUwMUqx_g) thread (second mail) and since sqlite is able to load extension.

Yes, the created base can be used use outside the plugin (with spatialite command for instance)